frontend/src/metabase/query_builder/components/view/ViewHeader/components/SavedQuestionLeftSide/util.ts

```diff
@@ -0,0 +1,21 @@
+import { useGetCardQueryMetadataQuery } from "metabase/api";
+import * as Lib from "metabase-lib";
+import type Question from "metabase-lib/v1/Question";
+import type { Table } from "metabase-types/api";
+
+export function useHiddenSourceTables(question: Question): Table[] {
```

Comment:
Usually people use either `hooks.ts` or just call it by the hook name, e.g. `use-hidden-source-tables.ts`. `utils` are for regular functions only.